spawn.lua

```diff
@@ -1,66 +1,45 @@
+function SendToWorldSpawn( Player )
+
+	local World = Player:GetWorld()
+	Player:TeleportToCoords(World:GetSpawnX(), World:GetSpawnY(), World:GetSpawnZ())
```

Comment:
The `TeleportToCoords` function still needs a valid destination chunk, so this code should actually use a `ChunkStay`.